input/postgres/relations.go

```diff
@@ -28,6 +30,8 @@ const relationsSQL string = `
 				c.reltoastrelid IS NOT NULL AS relation_has_toast,
 				c.relfrozenxid AS relation_frozen_xid,
 				%s,
+				COALESCE((SELECT inhparent FROM pg_inherits WHERE inhrelid = c.oid ORDER BY inhseqno LIMIT 1), 0) AS parent_relid,
```

Comment:
I'm assuming you gave the multiple-parent situation some thought--does it not make sense to support that?

Reply:
Yeah, its a good question - I thought some about it, and did it this way to keep things simple and performant on the pganalyze application side.

I think cascading hierarchies (i.e. having a parent that also has a parent) are much more common compared to having multiple inheritance. For one, multiple inheritance wouldn't be supported with declarative partitioning, i.e. its only a use case when you use the old-style table inheritance. We have one such use case in our own codebase, but that is for legacy reasons, and not because we actually use it.

Supporting multiple inheritance end-to-end (i.e. including in our own internal code) is certainly more expensive, and I think not warranted at this point, based on anecdotal evidence.

Therefore, I'll leave this as is for now - we can always adjust the data model here in the future if there is sufficient demand.